chapter08/VariousMaze.py

```diff
@@ -5,7 +5,7 @@
 #######################################################################
 
```

Comment:
line 153 and line 155 will crash under python3. The cause is [this](http://stackoverflow.com/questions/17322668/typeerror-dict-keys-object-does-not-support-indexing)